docassemble/assemblylinewizard/interview_generator.py

```diff
@@ -315,14 +319,10 @@ def source(self, follow_additional_fields=True):
         elif self.type == 'interview order':
             # TODO: refactor this. Too much of it is assembly-line specific code
             # move into the interview YAML or a separate module/subclass
-            content += "id: " + self.interview_label + "\n"
+            content += "id: interview_order_" + self.interview_label + "\n"
```

Comment:
Were we calling this the logic control or logic block or something? I think we need to pick a canonical name.